tests/ci/run_posix_tests.sh

```diff
@@ -39,7 +39,14 @@ fi
 
 # Lightly verify that uncommon build options does not break the build. Fist
 # define a list of typical build options to verify the special build option with
-build_options_to_test=("" "-DBUILD_SHARED_LIBS=1" "-DCMAKE_BUILD_TYPE=Release" "-DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release")
+TEST_SYSGENID_PATH=$(mktemp)
+build_options_to_test=(
+    ""
+    "-DBUILD_SHARED_LIBS=1"
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release"
+    "-DTEST_SYSGENID_PATH='${TEST_SYSGENID_PATH}'"
```

Comment:
does this pass through memory testers?

Reply:
In fact, this doesn't actually run any tests AFAIK

Reply:
I think all of Linux-based our CodeBuild jobs use it.